src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -291,7 +321,7 @@ bool ansi_c_entry_point(
   const code_typet::parameterst &parameters=
     to_code_type(symbol.type).parameters();
 
-  if(symbol.name==standard_main)
+  if(symbol.name==ID_main)
```

Comment:
Yes, the use use of the standard_main parameter looks  a bit dodgy. The only places where it is used are in src/cpp/cpp_language.cpp and src/goto-cc/compile.cpp, and there "main" is passed... So, it's not entirely clear why this parameter is needed at all.